app/src/ui/branches/branch-list-item-context-menu.tsx

```diff
@@ -0,0 +1,35 @@
+import { IMenuItem } from '../../lib/menu-item'
+import { clipboard } from 'electron'
+
+export function createBranchContextMenuItems(
+  name: string,
+  isLocal: boolean,
+  onRenameBranch?: (branchName: string) => void,
+  onDeleteBranch?: (branchName: string) => void
+): IMenuItem[] {
+  const items: Array<IMenuItem> = []
```

Comment:
We typically use `new Array<T>` instead of `Array<T> = []`
```suggestion
  const items = new Array<IMenuItem>()
```